### ISSUE TITLE
Does not work on linux due to msvrct.getch

### ISSUE THREAD

**Issue by SyntaxVoid** [SyntaxVoid/py_menu#1]
Will crash on Linux when importing since getch is only available on Windows. 

Solution should be something like:

IF OperatingSystem == Windows:
    THEN import msvrct.getch
ELSE:
    def getch(): ...

**Comment by SyntaxVoid** [comment]
Maybe this will fix it - Replace `import msvrct.getch` in `__init__.py` with the following code block? I don't have access to Linux for a while so I can't test but someone can create a branch, test, and submit a pull request? Easy contribution points
```python
import sys

try:
    from msvrct import getch
except ImportError:
    try:
        import tty, termios
        def getch():
            fd = sys.stdin.fileno()
            old = termios.tcgetattr(fd)
            try:
                tty.setraw(fd)
                ch = sys.stdin.read(1)
            finally:
                termios.tcsetattr(fd, termios.TCSADRAIN, old)
            return ch
    except:
        getch = input
except:
    getch = input
```

**Comment by SyntaxVoid** [comment]
Patched into version 1.0.2. (Commit # 75eefa1fabee2abb934f7507e47d57286c3997f9)
